### PR TITLE
[ZH] Add `Beatmap_ranking_procedure`

### PR DESCRIPTION
**Pull request by YumeMuzi**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Beatmap_ranking_procedure/zh.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+每个游戏模式获得 2 个提名后，谱面即进入了[过审 (Qualified)](/wiki/Beatmap/Category#qualified) 状态。已过审谱面有分数排行榜，但玩家游玩时不会获得任何[表现分 (pp)](/wiki/Performance_points)。当谱面处于过审状态并在[上架队列](Ranking_queue)中排队至少 7 天后，即可按队列进入[上架 (Ranked)](#ranked) 状态。
+
+过审状态存在的意义是把谱面推给社区，从而找出谱面中可能存在的隐藏问题。玩家可以从过审谱面的谱面讨论页向 BN 或者 NAT 反映谱面问题。与待定 (Pending) 谱面不同，谱师不能在谱面状态不发生改变的前提下，再次提交以修改这张谱面。若要修改谱面，谱师只能先请求[重置提名](#nomination-resets)，然后听取其他人意见或建议来修改它。
+
+## 提名重置
+
+提名重置能确保摸图员、BN 和 NAT 成员在谱面进入上架队列前，能检查到已排除问题、最新版本的谱面。
+
+当谱师提交更新了过审状态的谱面时，系统会立即重置谱面的提名。当 BN 或 NAT 发现过审谱面内的问题，或是[全局管理团队 (GMT)](/wiki/People/The_Team/Global_Moderation_Team) 成员发现谱面内存在不适当的内容时，也可以重置提名。谱面一旦重置提名就会立即下架 (Disqualified，或 DQ)，同时从上架队列内移除。
+
```
Comment (Walavouchey) [inline wiki/Beatmap_ranking_procedure/zh.md:36]: missed a paragraph?

> Members of the BN and NAT occasionally [veto](https://github.com/ppy/osu-wiki/blob/master/wiki/People/The_Team/Beatmap_Nominators/Beatmap_Veto) beatmaps they believe have quality issues that are unrelated to the Ranking Criteria.

Comment (YumeMuzi) [reply wiki/Beatmap_ranking_procedure/zh.md:36]: oops

File: wiki/Beatmap_ranking_procedure/zh.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+谱面作者（以下简称谱师）能够按照以下的谱面审核流程制作[谱面](/wiki/Beatmap)，即可让谱面被社区推选并进入[上架 (Ranked)](/wiki/Beatmap/Category#ranked) 状态。
+
+## 摸图
+
+谱师能在提交谱面时标记谱面为 `制作中 (Work in Progress)` 或`待定 (Pending)` 状态。这两种状态表示谱面开放接受社区的意见和建议。
+
+[摸图](/wiki/Modding)指接受与谱面相关的建设性批评的过程，这有助于提高谱面质量。玩家通常会在谱面讨论页发表意见和建议，也能直接与谱师私聊讨论谱面中可能存在的问题。
+
+若经验不足的新谱师想要让自己的谱面质量达到上架标准，则通常需要大量的摸，有时候甚至需要推倒重来 (Remap)。
```
Comment (kj415j45) [inline wiki/Beatmap_ranking_procedure/zh.md:13]:
```suggestion
经验不足的新谱师想要让自己的谱面质量达到上架标准，往往需要大量的修改，有时候甚至需要推倒重来 (Remap)。
```

File: wiki/Beatmap_ranking_procedure/zh.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+在通过审核并上架之前，谱面通常都需要至少获取 5 个来自不同玩家的[推荐](/wiki/Beatmap/Hype)。
+
+## 提名
+
+[谱面审核员 (BN)](/wiki/People/The_Team/Beatmap_Nominators) 的摸图经验丰富，并且能提名谱面。他们的职责是确保谱面的质量既能达到[上架标准 (Ranking Criteria)](/wiki/Ranking_Criteria)，也能达到他们个人的质量标准。
+
+[审核评估团队 (NAT)](/wiki/People/The_Team/Nomination_Assessment_Team) 成员也能提名谱面。但这不是他们的主要职责。
```
Comment (kj415j45) [inline wiki/Beatmap_ranking_procedure/zh.md:21]: BN 和 NAT 的译法与之前的译法不一致（？）

Comment (YumeMuzi) [reply wiki/Beatmap_ranking_procedure/zh.md:21]: NAT 是对的 BN 漏了一个“成”字 加上了

**Review by kj415j45 — DISMISSED** [review]

File: wiki/Beatmap_ranking_procedure/zh.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+建议在谱面已获得足够的摸和过审前必需的 5 个推荐后，再去请求 BN 提名谱面（俗称点图）。
+
+## 过审
+
+每个游戏模式获得 2 个提名后，谱面即进入了[过审 (Qualified)](/wiki/Beatmap/Category#qualified) 状态。已过审谱面有分数排行榜，但玩家游玩时不会获得任何[表现分 (pp)](/wiki/Performance_points)。当谱面处于过审状态并在[上架队列](Ranking_queue)中排队至少 7 天后，即可按队列进入[上架 (Ranked)](#ranked) 状态。
+
+过审状态存在的意义是把谱面推给社区，从而找出谱面中可能存在的隐藏问题。玩家可以从过审谱面的谱面讨论页向 BN 或者 NAT 反映谱面问题。与待定 (Pending) 谱面不同，谱师不能在谱面状态不发生改变的前提下，再次提交以修改这张谱面。若要修改谱面，谱师只能先请求[重置提名](#nomination-resets)，然后听取其他人意见或建议来修改它。
+
+当 BN 或者 NAT 成员认为谱面含有某些与上架标准 (Ranking Criteria) 无关的质量问题时，偶尔也会[否决 (veto)](/wiki/People/The_Team/Beatmap_Nominators/Beatmap_Veto) 谱面。
+
```
Comment (Walavouchey) [inline wiki/Beatmap_ranking_procedure/zh.md:32]: wrong line

```suggestion
```

File: wiki/Beatmap_ranking_procedure/zh.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+## 提名重置
+
+提名重置能确保摸图员、BN 和 NAT 成员在谱面进入上架队列前，能检查到已排除问题、最新版本的谱面。
+
+当谱师提交更新了过审状态的谱面时，系统会立即重置谱面的提名。当 BN 或 NAT 发现过审谱面内的问题，或是[全局管理团队 (GMT)](/wiki/People/The_Team/Global_Moderation_Team) 成员发现谱面内存在不适当的内容时，也可以重置提名。谱面一旦重置提名就会立即下架 (Disqualified，或 DQ)，同时从上架队列内移除。
+
```
Comment (Walavouchey) [inline wiki/Beatmap_ranking_procedure/zh.md:38]:
```suggestion

当 BN 或者 NAT 成员认为谱面含有某些与上架标准 (Ranking Criteria) 无关的质量问题时，偶尔也会[否决 (veto)](/wiki/People/The_Team/Beatmap_Nominators/Beatmap_Veto) 谱面。

```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**